src/common/snippets/src/port_descriptor.cpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/port_descriptor.hpp"
+#include "ngraph/except.hpp"
+#include <sstream>
+
+namespace ngraph {
+namespace snippets {
+
+size_t PortDescriptor::Scheduling::FULL_DIM = SIZE_MAX;
```

Comment:
Why `LOOP_NULL_ID` is defined in the expression, but `FULL_DIM` here? Shouldn't `LOOP_NULL_ID` also be in `Scheduling::` ?

Reply:
Need to discuss it offline because I think..

- `FULL_DIM` is about `PortDescriptor`
- `LOOP_NULL_ID` is about loop marking

these are different things

Reply:
Agreed to leave as is. But don't forget to rename `struct Scheduling` in `PortDescriptor`

Reply:
Thanks a lot for the remainder! I forgot about that. Renamed